codebuild/bin/install_default_dependencies.sh

```diff
@@ -30,7 +30,7 @@ if [[ "$TESTS" == "fuzz" || "$TESTS" == "ALL" ]]; then
 fi
```

Comment:
Should we also remove references to integv1 from Makefile, tests/Makefile?

Reply:
that can be done in a followup PR that deletes all of integv1